README.md

```diff
@@ -15,4 +15,13 @@ results you would get.
 Also give a theoretical argument for why X could or could not be correct, based
 on the complexity of the general sorting problem we covered in class.
 
+I would test the sorting by giving it small array sizes to start with like 1,2,3 to get a base case for time since it claims to work linearly. 
+Next I would test it with larger array sizes of 10^n and check if the sorting is still running at O(n). Finally I would check worst cases like
+having the array backwords. If the array time isn't the same reguardless of best case and the worst case then the run time complexity must not be O(n).
+
+If you look at the complexity from a theoretical view point then it is increadibly unlikely as the best case for most sorting alorithms is O(nlogn).
```

Comment:
And what about the argument we made for the theoretical complexity of the sorting problem?